app/controllers/v0/message_drafts_controller.rb

```diff
@@ -46,5 +46,12 @@ def draft_params
     def reply_draft_params
       @reply_draft_params ||= params.require(:message_draft).permit(:body)
     end
+
+    def history?(id)
```

Comment:
why are we doing this in the controller as opposed to lib/api methods? I don't think this belongs in the controller. We should build up the json using the same sort of thing we did for the multiple requests to build a collection, or for prescriptions tracking for example. This should be hidden away from view in the controller...